manifests/profile/python.pp

```diff
@@ -16,7 +16,14 @@
 #  include st2::profile::python
 #
 class st2::profile::python {
-  if !defined(Class['::python']) {
+  if $::osfamily == "RedHat" {
+    package {'python27':
```

Comment:
This isn't looking good for trying to use the python module to install 2.7 on EL6:
https://github.com/stankevich/puppet-python/blob/master/manifests/params.pp#L15